ros_ign_bridge/README.md

```diff
@@ -28,13 +28,15 @@ service calls. Its support is limited to only the following message types:
 | geometry_msgs/msg/Twist              | ignition::msgs::Twist                |
 | mav_msgs/msg/Actuators (TODO)        | ignition::msgs::Actuators (TODO)     |
 | nav_msgs/msg/Odometry                | ignition::msgs::Odometry             |
+| rcl_interfaces/msg/ParameterValue    | ignition::msgs::Any                  |
 | ros_ign_interfaces/msg/Contact       | ignition::msgs::Contact              |
 | ros_ign_interfaces/msg/Contacts      | ignition::msgs::Contacts             |
 | ros_ign_interfaces/msg/Dataframe     | ignition::msgs::Dataframe            |
 | ros_ign_interfaces/msg/Entity        | ignition::msgs::Entity               |
 | ros_ign_interfaces/msg/GuiCamera     | ignition::msgs::GUICamera            |
 | ros_ign_interfaces/msg/JointWrench   | ignition::msgs::JointWrench          |
 | ros_ign_interfaces/msg/Light         | ignition::msgs::Light                |
+| ros_ign_interfaces/msg/ParamVec      | ignition::msgs::Param                |
```

Comment:
hmm should we just call this `ros_ign_interfaces/msg/Param[s]`? or will `ignition::msgs::Param_V` be mapped to a type called `ros_ign_interfaces/msg/ParamVecVec`?

Reply:
My plan was to map Param_V to the same structure, but flatten it out, so that entries will become:

```param_0/foo
param_1/bar
etc```

Does this seem reasonable?